### DISCUSSION TITLE
Inclusion of OKX Wallet in the wallet list

### DISCUSSION THREAD

**Discussion by 0xAlexander** [WalletConnect/walletconnect-monorepo#3531] [Developer Support]
Hey team! @0xAsimetriq @glitch-txs 

We have to integrate OKX Wallet and noticed it's not in web3modal's default list. We've been using the defaults for maintainability and would prefer not to start defining each wallet manually if possible.

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/105828283/10613e63-3b96-4cfb-85e4-eeeeeb2cb3ac)

Considering that OKX Wallet is available and has a [wallet ID](https://walletconnect.com/explorer/okx-wallet), is it possible to consider adding OKX to the default list when wallets are undefined? It'd help us and possibly many others.

Thanks!

**Comment by glitch-txs** [comment]
Hi, at the moment only wallets that have desktop version are shown on desktop, we're improving this on [web3modal v3](https://docs.walletconnect.com/2.0/web3modal/v3/about)

[Alpha demo](https://web3modal-laboratory-v3.vercel.app/)

**Reply by 0xAlexander** [reply]
Thank you very much!

**Reply by 0xAlexander** [reply]
Oh by the way, OKX has a browser extension now! So i think they good for PC too?

**Reply by glitch-txs** [reply]
I didn't know about that, they need to register, we'll get in touch, thanks for pointing out 🤝

**Reply by 0xAlexander** [reply]
🤝

**Comment by 0xAlexander** [comment]
> I didn't know about that, they need to register, we'll get in touch, thanks for pointing out 🤝

Any update on this?
